src/ScaleLoader.tsx

```diff
@@ -19,8 +19,9 @@ function ScaleLoader({
   width = 4,
   radius = 2,
   margin = 2,
+  barCount = 5,
   ...additionalprops
-}: LoaderHeightWidthRadiusProps): JSX.Element | null {
+}: LoaderHeightWidthRadiusProps & { barCount: number }): JSX.Element | null {
```

Comment:
This should be an optional property: `{ barCount?: number }`.
v0.16 is a breaking change because now all uses of <ScaleLoader/> require barCount to be specified.

Reply:
@msheakoski good catch, bumped a fix to v0.16.1, and marked v0.16.0 as deprecated